tests/helper/test_helper.py

```diff
@@ -539,6 +540,9 @@ def test_download_media(
     ) = media_path
     controller = mocker.Mock()
 
+    if PLATFORM == "WSL":
+        media_path = media_path.replace("/", "\\")
+
```

Comment:
I'm not sure if I can import PLATFORM into tests, but this seemed like the only way to address the difference in file paths in WSL.

Reply:
To further clarify, based on a discussion on CZO, the issue stems from the `normalized_path` function used in the `download_media` method. The `media_path` assertion doesn't account for the WSL-specific normalized path.

Reply:
For a conditional in a test, the cleaner way to handle this would normally be to parametrize the test (over PLATFORM), though note you'd want a third path for the `expected_return_value` or similar output parameter, since it doesn't correspond to `url` or `media_path` in every case (though the latter may not be representative - see below re confirming).

The alternative to this conditional/parametrizing solution, which may also be clearer in the long term, is related to how (as we confirmed in the channel) the path returned is the path used for *opening* the temporary file, not the path used for *saving* it. So it would be clearer to refactor the code before fixing the tests, such that the `normalized_file_path` occurs outside of this function, and this function would be platform-independent to test.

Reply:
Moved the `normalized_file_path` to be called in `process_media` 👍 
 Clear explanation, thanks!